packages/features/portfolio/src/containers/portfolio-state-zero/hooks/use-card-data.tsx

```diff
@@ -95,16 +104,26 @@ export const useCardData = (
 
   const leveragedData = [
     {
```

Comment:
I like doing with this array of objects because it makes it easy to update the values in the future. If I pass down the yieldData this formatting all of this data is still going to have to happen just at the card level.